content/management/configuration-api/v3.3/index.mdx

```diff
@@ -680,6 +680,291 @@ curl -X POST \
+
+Deletes an existing `auto access` data structure specified by its ID.
+
+#### Specifics
+
+|                |                                                                                  |
+| -------------- | -------------------------------------------------------------------------------- |
+| Method URL     | `https://api.livechatinc.com/v3.3/configuration/action/delete_auto_access`       |
+| Required scopes| `access_rules:rw`                                                                |
+
+
+| Parameter              | Required     | Data type | Notes                         |
+| ---------------------- | ------------ | --------- | ----------------------------- |
+| `id`                   | Yes          | `string`  | Auto access ID                |
+
+#### Response
+
+No response payload (`200 OK`).
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.3/configuration/action/delete_auto_access \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -d '{
+    "id": "pqi8oasdjahuakndw9nsad9na"
+  }'
+```
+
+</CodeSample>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Reorder Auto Access
+
+Moves an existing `auto access` data structure, specified by `id`, before another one, specified by `next_id`. 
+If `next_id` is empty, then the auto access will be placed at the end of the list.
+
+#### Specifics
+
+|                |                                                                                  |
+| -------------- | -------------------------------------------------------------------------------- |
+| Method URL     | `https://api.livechatinc.com/v3.3/configuration/action/reorder_auto_access`      |
+| Required scopes| `access_rules:rw`                                                                |
+
+
+| Parameter              | Required     | Data type | Notes                                                                                                                                    |
+| ---------------------- | ------------ | --------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`                   | Yes          | `string`  | ID of the auto access to move                                                                                                            |
+| `next_id`              | No           | `string`  | ID of the auto access that should follow the moved auto access. Leave empty if the moved auto access should be the last one on the list. |
+
+#### Response
+
+No response payload (`200 OK`).
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.3/configuration/action/reorder_auto_access \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -d '{
+    "id": "pqi8oasdjahuakndw9nsad9na"
```

Comment:
Maybe example should actually include the next_id?

Reply:
Good idea 👆🏻

Reply:
In each of the methods the `next_id` param is optional, and I used it in other examples, so I figured I'd leave one out ;) But no problem, I can add it here too.

Reply:
Fixed nao